services/wallet/router/router_v2.go

```diff
@@ -417,6 +418,17 @@ func validateInputData(input *RouteInputParams) error {
 	return nil
 }
 
+func (r *Router) SuggestedRoutesV2Async(input *RouteInputParams) {
+	r.scheduler.Enqueue(async.TaskType{
+		ID:     int64(0),
+		Policy: async.ReplacementPolicyCancelOld,
+	}, func(ctx context.Context) (interface{}, error) {
+		return r.SuggestedRoutesV2(ctx, input)
+	}, func(result interface{}, taskType async.TaskType, err error) {
+		signal.SendWalletEvent(signal.SuggestedRoutes, result)
```

Comment:
handle `err`

Reply:
Also, I don't think it's required now since, but in case you feel it's convenient to add it: with these Async API endpoints I think it's useful to have some `uuid` that's set by the caller that's included as part of the response (so you can know on the client side which Event matches which request).

Reply:
> We normally include an error code in the event's payload

Yes, I noticed that while testing the changes, thanks for the heads up.

> I think it's generally useful to have some uuid

In this context it's not explicitly needed, but maybe a good time to add it for future needs and possibly give an option to clients for more flexible implementation.